navi/src/main/java/com/trello/navi/model/ViewCreated.java

```diff
@@ -0,0 +1,39 @@
+package com.trello.navi.model;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.view.View;
+
+public class ViewCreated {
```

Comment:
There should be some newlines between stuff, to match formatting of the other models.